test/IntegrationTests/MSTest.Acceptance.IntegrationTests/DynamicDataTests.cs

```diff
@@ -11,7 +11,8 @@ namespace MSTest.Acceptance.IntegrationTests;
 public class DynamicDataTests : AcceptanceTestBase
```

Comment:
Can you please rename the file/class into `ParameterizedTestTests` or ` DataTestTests`? You are now testing more than just `DynamicData`.

Reply:
Updated,